bundler/lib/bundler/cli/common.rb

```diff
@@ -14,6 +14,22 @@ def self.print_post_install_message(name, msg)
       Bundler.ui.info msg
     end
 
+    def self.output_fund_metadata_summary(gems_seeking_funding)
+      current_dependencies = Bundler.definition.current_dependencies.map(&:name)
+      current_specs = Bundler.definition.specs
+
+      count = current_dependencies.count do |dep|
+        return 0 if (dep = current_specs[dep].first).nil?
```

Comment:
Can this `nil?` actually happen? Also, even if it could happen, I guess you wouldn't want to early `return 0`, but instead make the result of the block invokation to be `false`? If we can't find a test case where this is needed, I'd rather to make the `nil?` check at all.

Reply:
It can `nil`, but only in the test suites—I had tests failing without this check. I no longer have access to those runs, but perhaps they've been kept in the GitHub Actions logs? I guess what I could/should have done was to just fix the test suite. What do you think?

Reply:
Mmm, not sure, maybe undo the change and let's see what fails now.